### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by asnganje**
- Created dynamic page
- Included javascript file to store information 
- Added popup window

### CODE REVIEW

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @asnganje , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has commits from one person and its a group project, kindy ensure everybody contributes to the project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @asnganje, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has done individually and this is a pair programming project. Kindly inform student success. I cannot do anything I am sorry. I need to follow the senior code reviewer and I got the same suggestion. Kindly inform student success so you can move forward

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -54,40 +57,18 @@ <h2 class="support-text">I’am a Software Developer</h2>
...
+        </div>
+        <div id="recentProjectWorks"></div>
     </section>
-
     <section class="project-portfolio">
         <div class="mediacss-grid">
             <div class="gitem1">
-                <button class="buttongrid1" type="button">See Project</button>
+                <button class="buttongrid1 button2" type="button">See Project</button>
             </div>
```
Comment (mugishasam123) [inline index.html:72]: - [x] Please try to stick to Figma design as much as possible to make your work look professional .so you can try to leverage the positioning of items in the popup modal to match the design (fonts,colors, margins,padding,etc)

your design:
image

Figma design:
image

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @asnganje  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
alt
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors 💯 
- Github workflow was properly used 👍 , keep rocking 🚀 


## Required Changes ♻️
_Check the comments under the review._

- [x] Please try to fix the see project buttons in other projects' cards so that you can make your portfolio more engaging and professional. it seems they are not showing up and working .you got this 👍 

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modal.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
+const myMultipost = document.getElementById('portfolioModal');
+
+const myRecentWorks = [
```
Comment (Ismailco) [inline modal.js:3]: - [ ] [OPTIONAL] You can add more projects to the list so your project's popup is different when clicking on different projects

File: index.html [hunk 3]
```diff
@@ -54,158 +57,76 @@ <h2 class="support-text">I’am a Software Developer</h2>
...
-                <li>bootstrap</li>
-                <li>Ruby</li>
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
                 </ul>
+                <button class="buttongrid1 button2" type="button">See Project</button>
             </div>
         </div>
         <div class="grid-container2">
```
Comment (Ismailco) [inline index.html:129]: - [ ] [OPTIONAL] The project section should be created dynamically using JavaScript, kindly implement this so your project will be complete

**Review by Ismailco — APPROVED** [review]
### Hi @asnganje, @LuisDavidRodriguez,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**